src/scripts/db_script.sql

```diff
@@ -0,0 +1,3 @@
+-- DATABASE : ea_restaurant_db
+
+CREATE DATABASE ea_restaurant_db
```

Comment:
Add EOF.

Also add the create table script here.

Reply:
done